src/exec_from.cr

```diff
@@ -1,15 +1,27 @@
+require "json"
+
 module ExecFrom
+  struct Result
+    include JSON::Serializable
+
+    getter status : Process::Status
```

Comment:
_If_ the JSON serialisation is going here, this may need to dump out the exit code.

Reply:
I don't think `to_json` is really required for this object. it's not used anywhere for that purpose.
if needed, we'll put it together.